src/apps/allbridge/shortcuts.e2e.ts

```diff
@@ -0,0 +1,34 @@
+import hook from './shortcuts'
+import { NetworkId } from '../../types/networkId'
+
+describe('getShortcutDefinitions', () => {
+  it('should get the definitions successfully', async () => {
+    const shortcuts = await hook.getShortcutDefinitions(
+      NetworkId['arbitrum-one'],
+    )
+    expect(shortcuts.length).toBeGreaterThan(0)
```

Comment:
Should we check for the expected shortcut IDs here?

Reply:
Added in e9dba5f2d289a26da9f6f01d7f9b89d0c7f2707e!